app/src/main/java/com/framgia/soundclound/screen/local/TrackLocalViewModel.java

```diff
@@ -0,0 +1,71 @@
+package com.framgia.soundclound.screen.local;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.ContentResolver;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.database.Cursor;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.net.Uri;
+import android.provider.MediaStore;
```

Comment:
ERROR: Unused import: android.provider.MediaStore. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports